src/components/ParticipantItem/ParticipantItem.js

```diff
@@ -1,128 +1,35 @@
-      this.setState({
-        messageOpenStatus: true,
-        message: "Name must not be empty and money must not be less than 0"
-      });
-      return;
-    }
-    //Ensuring there is no duplicating participant
-    this.props.participants.forEach(participant => {
-      if (this.state.participantName === participant.participantName) {
-        this.setState({
-          messageOpenStatus: true,
-          message: "There can not be two participant with the same name"
-        });
-        return;
-      }
-    });
-
-    let newParticipant = {
-      participantName: this.state.participantName,
-      money: this.state.money,
-      description: this.state.description
-    };
-
-    this.props.handleInput(newParticipant);
-    this.setState({
-      messageOpenStatus: true,
-      message: "Data added!"
-    });
-  };
-
-  handleClose = () => {
-    this.setState({ messageOpenStatus: false });
-  };
-
-  render() {
-    return (
       <div>
-        <FormControl style={{ borderStyle: "solid", borderColor: "blue" }}>
-          <TextField
-            required
-            name="participantName"
-            label="Name"
-            type="text"
-            onChange={this.handleChange}
-            value={this.state.participantName}
-          />
-          <TextField
-            required
-            name="money"
-            label="Spent money"
-            type="number"
-            onChange={this.handleChange}
-            value={this.state.money}
-          />
-          <TextField
-            name="description"
-            label="Description"
-            type="text"
-            onChange={this.handleChange}
-            value={this.state.description}
-          />
-          <div>
-            <Button
-              variant="contained"
-              color="primary"
-              style={{
-                width: 15,
-                margin: "auto",
-                marginTop: 15,
-                backgroundImage:
-                  "linear-gradient(to right bottom, #2196f3, #2985e5, #3174d6, #3962c6, #3f51b5)"
-              }}
-              onClick={this.handleInput}
-            >
-              Save
-            </Button>
-          </div>
-        </FormControl>
-        <Snackbar
-          anchorOrigin={{
-            vertical: "bottom",
-            horizontal: "center"
-          }}
-          open={this.state.messageOpenStatus}
-          autoHideDuration={2000}
-          onClose={this.handleClose}
-          ContentProps={{
-            "aria-describedby": "message-id"
+        <Button
+          variant="contained"
+          color="primary"
+          style={{
+            width: 10,
+            margin: "auto",
+            marginTop: 15,
+            backgroundImage:
+              "linear-gradient(to right bottom, #2196f3, #2985e5, #3174d6, #3962c6, #3f51b5)"
           }}
-          message={this.state.message}
-        />
+        >
+          Edit
+        </Button>
```

Comment:
Let's remove the edit button for now, as it does nothing atm. We will discuss about this edit function later.